ludwig/models/predictor.py

```diff
@@ -116,6 +116,7 @@ def __init__(
         self.device = device
         self.dist_model = dist_model
         self.model = model
+        self.model.metrics_to_device(device)
```

Comment:
As much as I love 1-line bug fixes, it behooves us to leave a comment so that future readers (which includes ourselves) have more context for why this line of code is important.